gateway/api/services/storage/logs_storage.py

```diff
@@ -90,3 +95,40 @@ def get(self, job_id: str) -> Optional[str]:
                 str(e),
             )
             return None
+
+    def put(self, job: Job, log: str) -> None:
+        """
+        Write a log in a file for the given job id
+
+        Args:
+            job (Job): the job to write the logs
+            log (str): the job to write the logs
+
+        Returns:
+            None
+        """
+        log_path = self._get_logs_path(job.id)
+        max_mb = int(settings.FUNCTIONS_LOGS_SIZE_LIMIT)
+        file_size = 1024 * 1024 * 1
+
+        try:
+            file_logger.remove()
+            file_logger.add(
+                log_path,
+                rotation=file_size,
+                retention=math.trunc(max_mb / file_size),
```

Comment:
We need to study what configuration we want to use here taking into account the next use cases:
- If we create several files to retrieve those logs could be a bit of a pain (access to different files)
- What happens if a log is bigger than the retention / rotation policy.